src/scripts/main.js

```diff
@@ -1 +1,108 @@
 'use strict';
+
+let isClickedLeft = false;
+let isClickedRight = false;
+let timer;
+const notification = document.createElement('div');
+
+notification.setAttribute('data-qa', 'notification');
+
+function resetTimer() {
+  clearTimeout(timer);
+
+  timer = setTimeout(() => {
+    if (!isClickedLeft) {
+      firstPromise();
+    }
+  }, 3000);
+}
+
+function handleFirstClick() {
+  isClickedLeft = true;
+  firstPromise();
+  resetTimer();
+
+  document.removeEventListener('click', handleFirstClick);
+}
+
+document.addEventListener('click', handleFirstClick);
+
+document.addEventListener('mousedown', (e) => {
+  if (e.button === 0) {
+    isClickedLeft = true;
+  }
+
+  if (e.button === 2) {
+    isClickedRight = true;
+  }
+
+  if (isClickedLeft !== isClickedRight) {
+    secondPromise();
+  }
```

Comment:
The condition `isClickedLeft !== isClickedRight` will be true even if neither button was clicked (both variables are `false`). This might not be the intended behavior. Consider revising this condition to ensure that it accurately represents the desired logic.